modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -747,7 +771,8 @@ the board to make the detection more robust in various environments.
  */
 CV_EXPORTS_W bool findCirclesGrid( InputArray image, Size patternSize,
                                    OutputArray centers, int flags = CALIB_CB_SYMMETRIC_GRID,
-                                   const Ptr<FeatureDetector> &blobDetector = SimpleBlobDetector::create());
+                                   const Ptr<FeatureDetector> &blobDetector = SimpleBlobDetector::create(),
+                                   CirclesGridFinderParameters parameters = CirclesGridFinderParameters());
```

Comment:
This will not work and break ABI (binary and backwards compatibility). You need to make an overload function that accepts an extra parameter and leave the default functionality as is. Else your suggestion will never get merged in.